pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -548,3 +554,16 @@ func upsertObject(ctx context.Context, c ctrlruntimeclient.Client, obj crcontrol
 	}
 	return err
 }
+
+var deniedRegistries = sets.NewString("default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com",
```

Comment:
This list is guaranteed to expand over time. Can we make this configurable?

Reply:
Also, please add a testcase for this

Reply:
Done.